src/redux/actions.js

```diff
@@ -63,5 +64,12 @@ export const deleteTodo = (payload) => {
         payload: payload
     }
 }
+    
+export const judgeDoubleClick = (payload) => {
```

Comment:
this action definition has payload, but when you call this function, you did not pass any arguments to it.

Reply:
the name does not seems show what this function do. Need a better naming.